x-pack/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -873,4 +873,14 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'linux.advanced.kernel.capture_mode',
+    first_supported_version: '8.2',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.linux.advanced.kernel.capture_mode',
+      {
+        defaultMessage: 'Description needed. Default: true',
```

Comment:
The flag has three options ['auto', 'ebpf', 'kprobes']. 
Truthfully the pseudo code is 
if 'auto' : // autodetect platform for capture mode
else if 'ebpf' : // try ebpf first, otherwise fall back
else: // just default to kprobes

So for now the default state is `kprobes`

Reply:
What about something like `Control the method Endpoint uses to capture event data. Default is "auto" (Endpoint selects the best method), other options are "kprobe" or "ebpf" to force event capture using those methods.`?

That's a forward looking sentence, we haven't switched to auto mode yet but expect to.